content/blog/infer-1-0-0/index.Rmd

```diff
@@ -0,0 +1,429 @@
+---
+output: hugodown::hugo_document
+
+slug: infer-1-0-0
+title: infer 1.0.0
+date: 2021-08-16
+author: Simon Couch
+description: >
+    The first major release of infer, a package implementing a unified approach
+    to statistical inference, is now on CRAN.
+
+photo:
+  url: https://simonpcouch.com
+  author: Simon Couch
+
+categories: [package] 
+tags: [tidymodels]
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+We're super excited announce the release of [infer](https://infer.tidymodels.org/) 1.0.0! infer is a package for statistical inference that implements an expressive statistical grammar that adheres to the tidyverse design framework. Rather than providing methods for specific statistical tests, this package consolidates the principles that are shared among common hypothesis tests and confidence intervals into a set of four main verbs (functions), supplemented with many utilities to visualize and extract value from their outputs.
```

Comment:
Is it worth a sentence somewhere that this is specifically designed for teaching?